### PR TITLE
Fix reordering columns

### PR DESCRIPTION
**Pull request by OliverZiegler**
This PR fixes reordering the columns.

The former code lost the array key of the moved column so you couldn't remove/access it later on.

### CODE REVIEW

File: src/PanelTraits/Columns.php [hunk 1]
```diff
@@ -94,33 +94,37 @@ public function addColumns($columns)
     /**
      * Moves the recently added column to 'before' the $target_col.
      *
-     * @param $target_col
+     * @param $target string|array the name or array of the target column
```
Comment (lloy0076) [inline src/PanelTraits/Columns.php]: The name of the target column or an array containing the 'name' key? (Is there better wording than that?)

File: src/PanelTraits/Columns.php [hunk 2]
```diff
@@ -94,33 +94,37 @@ public function addColumns($columns)
...
+            array_slice($this->columns, $offset, null, true);
+
+        $this->columns = $array;
     }
 
     /**
      * Moves the recently added column to 'after' the $target_col.
      *
-     * @param $target
+     * @param $target string|array the name or array of the target column
```
Comment (lloy0076) [inline src/PanelTraits/Columns.php]: See above comment.

File: src/PanelTraits/Columns.php (hunk 1 above)
Comment (OliverZiegler) [reply src/PanelTraits/Columns.php]: Tried to be consistent with other php docs but can change according to your suggestion

**Comment by tabacitu** [comment]
@OliverZiegler I see @tumf87 has started a few PRs starting from your commits, so I'll close this and merge those. Let's collaborate on them :-)

Cheers!

**Comment by tabacitu** [comment]
Damn. Closed the wrong PR :-) Reopened.

**Comment by tabacitu** [comment]
@tumf87 what do you think about this? You found this problem too, right? Can you please test this solution and tell me if it works for you too?

**Comment by tabacitu** [comment]
@tumf87 am I wrong or has this issue been solved by your PR too, that's already merged into DEV?

Thanks, cheers!

**Comment by tumf87** [comment]
@tabacitu, yup, this was fixed by one of the PRs you merged.

I didn't looked into this PR because from the title I thought it was related to the `orderColumns` method (which also doesn't work), not the 'before' and 'after' ones.

But yeah, this can be closed now.

**Comment by tabacitu** [comment]
Great, closing! Thanks both of you for the hard work. Let's ship this thing!! Whoo-hoo!

**Pull request closed without merging**